frontend/Dockerfile

```diff
@@ -26,8 +36,10 @@ USER node
 WORKDIR /srv/bt
 COPY --from=build --chown=node:node /srv/bt/ .
 
-ENV ENV=prod
 ENV NODE_ENV=production
 
+# TODO: remove
+RUN cat common/project.js
```

Comment:
This outputs the correct `project_selfhosted.js` file contents, but then when I run the container, I still get the staging API key being used for the flagsmith on flagsmith API requests.

Reply:
This was user error 🤦